src/main/java/org/fundacionjala/sfdc/pageobjects/contacts/SFCNewModifyPage.java

```diff
@@ -0,0 +1,172 @@
+     * @param contactLastName string value.
+     */
+    public void setContactLastNameTextField(final String contactLastName) {
+        this.contactLastNameTextField.clear();
+        this.contactLastNameTextField.sendKeys(contactLastName);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactTitleTextField string value.
+     */
+    public void setContactTitleTextField(final String contactTitleTextField) {
+        this.contactTitleTextField.clear();
+        this.contactTitleTextField.sendKeys(contactTitleTextField);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactDepartmentTextField string value.
+     */
+    public void setContactDepartmentTextField(final String contactDepartmentTextField) {
+        this.contactDepartmentTextField.clear();
+        this.contactDepartmentTextField.sendKeys(contactDepartmentTextField);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactBirthdateTextField string value.
+     */
+    public void setContactBirthdateTextField(final String contactBirthdateTextField) {
+        this.contactBirthdateTextField.clear();
+        this.contactBirthdateTextField.sendKeys(contactBirthdateTextField);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactLeadSourceTextField string value.
+     */
+    public void setContactLeadSourceTextField(final String contactLeadSourceTextField) {
+        this.contactLeadSourceTextField.sendKeys(contactLeadSourceTextField);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactPhoneTextField string value.
+     */
+    public void setContactPhoneTextField(final String contactPhoneTextField) {
+        this.contactPhoneTextField.clear();
+        this.contactPhoneTextField.sendKeys(contactPhoneTextField);
+    }
+
+    /**
+     * Method to set contact las Name attribute.
+     * @param contactHomePhoneTextField string value.
+     */
+    public void setContactHomePhoneTextField(final String contactHomePhoneTextField) {
+        this.contactHomePhoneTextField.clear();
+        this.contactHomePhoneTextField.sendKeys(contactHomePhoneTextField);
+    }
+    /**
+     * Method to do click "save" button.
+     */
+    public void clickSaveContactButton() {
+        getSaveContactButton().click();
+    }
+    /**
+     * Method to do click "fillMethodsToFields" to map.
+     * @return map of contact fields.
+     */
+    public Map<String, Value> fillMethodsToFields() {
+        newContactsFields.put("contactName", (String v) -> {
+            setContactNameTextField(v);
+        });
+        newContactsFields.put("contactLastName", (String v) -> {
+            setContactLastNameTextField(v);
+        });
+        newContactsFields.put("contactTitle", (String v) -> {
+            setContactTitleTextField(v);
+        });
+        newContactsFields.put("contactDepartment", (String v) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)